.github/workflows/stale-issue.yml

```diff
@@ -0,0 +1,22 @@
+name: Close inactive issues
+on:
+  schedule:
+    - cron: "42 2 * * *"
+
+jobs:
+  close-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v5
+        with:
+          days-before-issue-stale: 30
+          days-before-issue-close: 14
```

Comment:
to clarify, the logic here is 30 days to mark as stale. after that designation, it's 14 additional days before the auto close? which is 👍

Reply:
aye, label after 30 days of idle, then close 14 days after label.

Reply:
thanks!